test/format/test_format.py

```diff
@@ -702,7 +704,7 @@ def test_makeboxes_text(str_expr, str_expected, form, msg):
     @pytest.mark.xfail
     def test_makeboxes_tex_fragile(str_expr, str_expected, form, msg):
         result = session.evaluate(str_expr)
-        format_result = result.format(session.evaluation, form)
+        format_result = format_element(result, session.evaluation, Symbol(form))
```

Comment:
Do we still need this? It might be good to leave at least one or two `result.format()` around since in my view this is the way I think it will be typically used.

Reply:
The aim of removing `format` from the interface of ``Element`` was to make it simpler, and avoid circular dependences. 

Conceptually, also I think that "formatting" is one of the possible many things you could do with an Expression. ("Analytical evaluation", "Numerical evaluation", or "Compilation" are other possible things). For this reason, I think that any of these things should be necessarily attached to the objects as methods. On the other hand, it does not harm to have them as methods if they are just called at a very high level. For example, having a complicated expression `expr`, it is fine to have 
a method `format`, that calls `format_element`, which internally works recursively. What I would like to avoid is having calls to the `format` method inside `format_element`, due that in some builtins that method is called. @rocky, makes this sense for you?

Reply:
I agree with what you write. To clarify a bit further. Wen we have something like `eval_N`, the top-level function is needed down the line because in an interpreter you wan a list of operations or instructions that are run, not methods in a class.  

In the unit test though, the purpose should be to test both forms of access, especially the method access because that's what will be used more often. 

In the code itself, it is easier to use the method from because that is "natural" and saves an import. It trades method lookup on an object for a import instruction. (Modern JITing pythons are supposed to drive down the cost of the method lookup.

Some other small things. Why is this test marked to fail? 

Personally, I would have iterated over Symbols rather than call the Symbol function inside the loop.  But these are minor things.

Reply:
> Some other small things. Why is this test marked to fail?

See the comment at the beginning of the test [/test/format/test_format.py L23](https://github.com/Mathics3/mathics-core/blob/30a5d875770e972304ac5bc410554d226907937c/test/format/test_format.py#L23). I can extend this comment in another PR if it is needed.


> 
> Personally, I would have iterated over Symbols rather than call the Symbol function inside the loop. But these are minor things.

OK